packages/terra-form-select/src/_Menu.jsx

```diff
@@ -313,19 +402,22 @@ class Menu extends React.Component {
 
   render() {
     return (
-      // This warns that aria-activedescendant should map to an id
-      // Our implementation maps it to a dynamic id but linter is unable to detect mapping and throws an error
-      /* eslint-disable jsx-a11y/aria-proptypes */
+      /**
+       * Note: role="listbox" and aria-activedescendant needed for VoiceOver on iOS to properly
+       * shift the virtual indicator to this DOM element when dropdown is rendered.
+       */
       <ul
+        id="terra-select-menu"
         role="listbox"
         className={cx('menu')}
         ref={(menu) => { this.menu = menu; }}
-        aria-activedescendant={`terra-select-option-${this.state.active}`}
+        {...('ontouchstart' in window ? { 'aria-activedescendant': `terra-select-option-${this.state.active}` } : {})}
+        data-activedescendant={`terra-select-option-${this.state.active}`}
         tabIndex="0"
+        style={{ outline: 'none' }}
```

Comment:
Could this be attached to the css class menu?

Reply:
Updated in this commit: https://github.com/cerner/terra-core/pull/2236/commits/7ed989860674b09c4c4c49b168d745eb37edc8f5